src/Planners/src/TimeVaryingDCMPlanner.cpp

```diff
@@ -491,6 +494,8 @@ struct TimeVaryingDCMPlanner::Impl
     bool setupOptimizationProblem(const ContactPhaseList& contactPhaseList,
                                   const DCMPlannerState& initialState)
     {
+        constexpr auto errorPrefix = "[TimeVaryingDCMPlanner::Impl::setupOptimizationProblem] ";
```

Comment:
Maybe we can add some tags to distinguish between errors and info messages.

Reply:
I would do it in a separate PR for all the error/warning/info in the repo. Indeed now all the errors/warnings/info are printed with `cerr` and there is not a common function to call.  https://github.com/dic-iit/bipedal-locomotion-framework/issues/137#issuecomment-736655938

Reply:
For instance, we can use something like this: https://github.com/robotology/idyntree/blob/2e5cf7180a517bf91777d48761c4c12c94e7e404/src/core/include/iDynTree/Core/Utils.h#L66-L93

Reply:
I was not referring to ``cerr`` but rather to the message itself. It is not clear whether the message is an error or not. I was suggesting adding something like ``[ERROR]`` since it is going to be much easier to find them in the middle of tons of other messages.

Reply:
On that one I agree done in https://github.com/dic-iit/bipedal-locomotion-framework/pull/220/commits/7999c2f0010291c9bda996d3a3b0eb218ce7c988